multicluster/service-mirror/cluster_watcher.go

```diff
@@ -192,19 +192,39 @@ func (rcsw *RemoteClusterServiceWatcher) originalResourceName(mirroredName strin
 	return strings.TrimSuffix(mirroredName, fmt.Sprintf("-%s", rcsw.link.TargetClusterName))
 }
 
-func (rcsw *RemoteClusterServiceWatcher) getMirroredServiceLabels() map[string]string {
-	return map[string]string{
+// Provides labels for mirrored service.
+// "remoteService" is an optional parameter. If provided, copies all labels from the remote service to mirrored service (except labels with the "SvcMirrorPrefix").
```

Comment:
very nitpick: we wrap our comments at col 80, would you please do the same here?

Reply:
Done!